src/main/scala/com/ubirch/webui/models/keycloak/group/GroupFactory.scala

```diff
@@ -3,30 +3,57 @@ package com.ubirch.webui.models.keycloak.group
 import com.typesafe.scalalogging.LazyLogging
 import com.ubirch.webui.config.ConfigBase
 import com.ubirch.webui.models.Exceptions.{ GroupNotFound, InternalApiException }
+import com.ubirch.webui.models.keycloak.tenant.Tenant
+import com.ubirch.webui.models.keycloak.tenant.Tenant.GroupToTenant
 import com.ubirch.webui.models.keycloak.util.{ GroupResourceRepresentation, Util }
 import com.ubirch.webui.models.keycloak.util.BareKeycloakUtil._
 import org.keycloak.admin.client.resource.GroupResource
 import org.keycloak.representations.idm.GroupRepresentation
 
+import javax.ws.rs.core.Response
 import scala.util.{ Failure, Success, Try }
 
 object GroupFactory extends LazyLogging with ConfigBase {
 
+  def getDefaultTenant: Tenant = {
```

Comment:
I am wondering if this func should be return a Try, it seems to me that it can fail and hide the error?